Automatically switches textmode if the user selects a wrong one.

Original author of the idea/first patch is unknown.

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
Signed-off-by: Luiz Fernando N. Capitulino <lcapitulino@mandriva.com.br>

--
 arch/x86/boot/video.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/x86/boot/video.c
+++ b/arch/x86/boot/video.c
@@ -325,8 +325,8 @@ void set_video(void)
 		if (!set_mode(mode))
 			break;
 
-		printf("Undefined video mode number: %x\n", mode);
-		mode = ASK_VGA;
+		printf("Undefined video mode number: %x, jump to 80x25\n", mode);
+		mode = VIDEO_80x25;
 	}
 	boot_params.hdr.vid_mode = mode;
 	vesa_store_edid();
